### ISSUE TITLE
Vagrant provisioner: /srv not on minions.

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/kubernetes#17797]
As per conv w/ mike in Slack... it looks like its possible that the /srv directory, which is needed for all docker images in the vagrant setup. However, its not being created in the minions...

Master looks ok...
```
[vagrant@kubernetes-master ~]$ ls /srv/
kubernetes  pillar  reactor  salt  salt-overlay  sshproxy
```
Nothing in here though ... 
```
[vagrant@kubernetes-minion-1 ~]$ ls /srv/
```

I haven't anything at all with the salt stuff, so Im not sure exactly when the provision-master.sh and provision-minion.sh have diverged.  

Looks like @erictune @mikedanese @derekwaynecarr would be the experts on this.

**Comment by davidopp** [comment]
Does this mean Vagrant is currently broken?

**Comment by jayunit100** [comment]
At first glance, yes.
- On my end , I did notice this .   I don't know if others did ? 
- As we all know, sometimes vbox or vagrant versions can be the culprit.

**Comment by mikedanese** [comment]
I think a good way to reduce drift between vagrant and gce would be to move vagrant to masterless salt

cc @derekwaynecarr

**Comment by derekwaynecarr** [comment]
@mikedanese @jayunit100 - So Vagrant is broken now after Thanksgiving?  What PR did I miss that broke it so I can react?

**Comment by mikedanese** [comment]
@derekwaynecarr  #17977 fixes this. There may be a faster way to fix this but #17977 reduces divergence between gce and vagrant which is a good think.

**Comment by jayunit100** [comment]
thanks for that patch . ill test it tonite
